GoogleSignIn/Tests/Unit/GIDSignInTest.m

```diff
@@ -738,136 +742,171 @@ - (void)testNotHandleWrongPath {
 
 #pragma mark - Tests - disconnectWithCallback:
 
+// Verifies the request url when disconnect succeeds.
+- (void)testDisconnect_verifyURL_succeed {
+  [_keychainHandler saveAuthState:_authState];
+  OCMStub([_authState lastTokenResponse]).andReturn(_tokenResponse);
+  OCMStub([_tokenResponse accessToken]).andReturn(kAccessToken);
+  
+  GIDHTTPFetcherTestBlock testBlock =
+      ^(GIDHTTPFetcherFakeResponse response) {
+        NSData *data = [[NSData alloc] init];
+        response(data, nil);
+     };
+  [_httpFetcher setTestBlock:testBlock];
+  
+  XCTestExpectation *expectation =
+      [self expectationWithDescription:@"Callback called with nil error"];
+  [_signIn disconnectWithCompletion:^(NSError * _Nullable error) {
+    XCTAssertNil(error);
+    NSURL *url = [self->_httpFetcher requestURL];
+    XCTAssertEqualObjects([url scheme], @"https", @"scheme must match");
+    XCTAssertEqualObjects([url host], @"accounts.google.com", @"host must match");
+    XCTAssertEqualObjects([url path], @"/o/oauth2/revoke", @"path must match");
+    OIDURLQueryComponent *queryComponent = [[OIDURLQueryComponent alloc] initWithURL:url];
+    NSDictionary<NSString *, NSObject<NSCopying> *> *params = queryComponent.dictionaryValue;
+    XCTAssertEqualObjects([params valueForKey:kSDKVersionLoggingParameter], GIDVersion(),
+                          @"SDK version logging parameter should match");
+    XCTAssertEqualObjects([params valueForKey:kEnvironmentLoggingParameter], GIDEnvironment(),
+                          @"Environment logging parameter should match");
```

Comment:
Should assert the token is in the post body too.

Reply:
Done.